crawler/lookingglass_crawler/proxy_refresher.py

```
@@ -0,0 +1,70 @@
+import time
+import logging
+import threading
```

Ruff (0.11.9)

`threading` imported but unused

Remove unused import: `threading`

(F401)